tests/suites/firewall/task.sh

```diff
@@ -8,19 +8,34 @@ test_firewall() {
 
 	setup_awscli_credential
 
+	setup_gcloudcli_credential
```

Comment:
We should move this and the aws cred setup down to the cloud specific sections below